src/lib/util.ts

```diff
@@ -14,23 +14,36 @@ const polyfills = {
 export const btoa = globalThis.btoa ?? polyfills.btoa;
 
 export const contractAddresses: { [index: string]: string } = {
```

Comment:
to be most consistent / clear, i think we should just match the naming pattern here: https://github.com/tablelandnetwork/eth-tableland/blob/main/hardhat.config.ts#L40

otherwise, "testnet" and "staging" are ambiguous

Reply:
I believe we can also leave out "optimism-kovan-staging" since that should be for internal use only. and since you changed the user API to be flexible, it would be easy for use to manually use this in our internal testing

Reply:
Agree that naming conventions should be consistent. But actually, we probably should have gone with these "standards": https://github.com/ethers-io/ethers.js/blob/master/packages/networks/src.ts/index.ts#L132. Many other projects use these apparently. For now, I'm happy just creating a mapping from what we're using to these internally (for the cli, these names are required).

Reply:
The above link references https://chainlist.org

Reply:
there's more information to capture though than what's in those names. ie, each tableland-to-chain pair, where the chain can be used twice, as we are doing with optimism kovan (and others in the future). for the CLI, maybe you can separate those, eg `--goerli --testnet` points to goerli on tableland testnet, `--goerli --staging` would point to goerli on tableland staging

Reply:
See my recent pushes. As far as the cli goes, its actually even more complicated than the above would suggest. We need to specify the contract address, _and_ the provider API to connect with. I can find some nice mappings to use, but not tonight :P

Reply:
Bumping this one again, just to make sure it looks like what y'all were expecting? In the CLI, essentially you have to specify via the "host" param if you are using staging or testnet, and it should pick the right one from that.

Reply:
taking a look